src/modules/formBuilder/components/itemEditor/conditionals/ManageEnableWhen.tsx

```diff
@@ -89,11 +89,7 @@ const ManageEnableWhen: React.FC<ManageEnableWhenProps> = ({
       {fields.map((condition, index) => (
         <RemovableCard
           key={JSON.stringify(condition)} // fixme could be non unique
-          onRemove={() => {
-            const isLastCondition = fields.length === 1;
-            remove(index);
-            if (isLastCondition) onChangeEnableBehavior(null);
-          }}
+          onRemove={() => remove(index)}
```

Comment:
Nice - it seems like this is no longer needed because the "Conditional Behavior" input gets unmounted when the last index is removed, is that right?

Reply:
Hm, no, I removed this because it was not working -- it seems to work instead with a `useEffect` which I mentioned [here](https://github.com/greenriver/hmis-frontend/pull/885#discussion_r1717121606). But, unmounting is also a good idea, let me try it out...

Reply:
Hrm! It has to do with the `useController` call to get the `onChangeEnableBehavior` handler in this file. For some reason, this is setting the enable behavior back to 'ALL' _after_ the component gets unmounted and sets it to `undefined`. Curiously, this happens even when the `onChangeEnableBehavior` call on line 61 is commented out.

Anyway, I _think_ removing the `useController` call and then adding a default value to the radio group input should achieve the desired behavior without interfering with the component's ability to unregister when it unmounts. Investigating...

Reply:
Ok, I see now that this behavior must've been written the way it was originally because RHF's controller's `defaultValue` doesn't quite work for this use case. Maybe related to its strict requirements around not passing `undefined`: https://react-hook-form.com/docs/usecontroller/controller

Anyway, passing `setValue` from `FormEditorItemProperties` does seem to work, if you think it's not too messy.

Reply:
Alternatively, we could remove this and leave it as a required field that doesn't get a default value.

Reply:
> For some reason, this is setting the enable behavior back to 'ALL' after the component gets unmounted and sets it to undefined.

Hmm, I can't reproduce this. To me the behavior looks correct. (When reverting to the useController approach). Maybe we can pair?

Reply:
oookay, summary:
- bug was reproducible by working from an item that was persisted with a condition (since that gets set as the default value)
- `useController`'s `onChange` is meant to only be used with onChange to an input component, it shoudl not be invoked manually (per docs) -- you are correct to use `setValue` for these instances
- there were 2 other places where we used useController hook in this way that may be prone to the same bug and should be replaced with setValue